src/background/services/zk-validator.ts

```diff
@@ -3,12 +3,17 @@ import { ZkInputs } from '@src/types'
 export default class ZkValidator {
     // eslint-disable-next-line class-methods-use-this
     validateZkInputs(payload: Required<ZkInputs>) {
-        const { circuitFilePath, zkeyFilePath, merkleStorageAddress, merkleProofArtifacts } = payload
+        const { circuitFilePath, zkeyFilePath, merkleStorageAddress, merkleProofArtifacts, merkleProof } = payload
```

Comment:
`merkleStorageAddress` seems to be unused here

Reply:
This has been fixed